sklearn/conftest.py

```diff
@@ -28,6 +29,29 @@
         "at least pytest >= {} installed.".format(PYTEST_MIN_VERSION)
     )
 
+scipy_datasets_require_network = sp_version >= parse_version("1.10")
+
+
+def raccoon_face_or_skip():
+    if scipy_datasets_require_network:
```

Comment:
Can you add a comment to always require network when scipy min version >= 1.10 ?